### ISSUE TITLE
Mixed precision model support for Vitis AI Quantizer

### ISSUE THREAD

**Issue by schegde** [Xilinx/Vitis-AI#573]
I have a PyTorch model trained with mixed precision enabled (float16 and float32), and I get `Exception has occurred: KeyError       (note: full exception trace is shown but execution is paused at: evaluate) 'half'`, when I pass the model through the vitis-AI quantizer API:  `quantizer = torch_quantizer(quant_mode, net, input_sample)` . I can confirm I dont get this error when I disable mixed precision in the input model. So:
1. Is there a plan to support mixed precision data types in future?
2. Would you expect quantization process to be very different with mixed precision disabled in model, and quantization proceeded with?

**Comment by niuxjxlnx** [comment]
Hi, @schegde :

So far vai_q_pytorch has no plan to support mixed precison float model input. 
To support this feature, vai_q_pytorch needs some data type support and conversion in flow, but not on quantization algorithm.

To make you model work, you can input a pure float32 model and quantize it with INT8, if the accuracy is not high enough, you can try fast finetune or QAT to improve the accuracy.

**Comment by schegde** [comment]
Thank you for the response!
